iet.so/src/iet_worker.cpp

```diff
@@ -176,26 +177,18 @@ void IETWorker::blasThread(int gpuIdx,  uint64_t matrix_size, std::string  iet_o
         iet_end_time = std::chrono::system_clock::now();
         //Duration in the call
         duration = time_diff(iet_end_time, iet_start_time);
-        gem_ops++;
 
         //Converting microseconds to seconds
         timetakenforoneiteration = duration/1e6;
         //calculating Gemm count
         gflops_interval = gpu_blas->gemm_gflop_count()/timetakenforoneiteration;
         //Print the gflops interval
         log_interval_gflops(gflops_interval);
+
         // check end test to avoid unnecessary sleep
         if (endtest)
             break;
-        //if gemm ops greater than 10000, lets yield
-        //if this is not happening we are ending up in
-        //out of memmory state
-        if(gem_ops > 10000) {
```

Comment:
why are we removing ?

Reply:
This sleep results in unsustained GPU power utilization resulting in fluctuating power usage which is not expected why testing max. power cap testing.